pkg/networkmanager/firewall.jsx

```diff
@@ -701,26 +702,27 @@ class ActivateZoneModal extends React.Component {
                    </>}
             >
                 <Form isHorizontal>
-                    <FormGroup label={ _("Trust level") }>
-                        <div role="group" className="add-zone-zones">
-                            <div className="add-zone-zones-firewalld">
-                                <legend>{ _("Sorted from least trusted to most trusted") }</legend>
+                    <FormGroup label={ _("Trust level") } className="add-zone-zones">
+                        <Flex>
+                            <FlexItem className="add-zone-zones-firewalld">
+                                <legend>{ _("Sorted from least to most trusted") }</legend>
                                 { zones.filter(z => firewall.predefinedZones.indexOf(z) !== -1).sort((a, b) => firewall.predefinedZones.indexOf(a) - firewall.predefinedZones.indexOf(b))
                                         .map(z =>
                                             <Radio key={z} id={z} name="zone" value={z}
-                                                   onChange={(value, e) => this.onChange("zone", e.target.value)}
-                                                   label={ firewall.zones[z].id } />
+                                                    defaultChecked={ firewall.predefinedZones.indexOf(z) === 1 }
```

Comment:
I couldn't get this working. I was trying to make the first default. I decided to give up and ask for you to help here.

So, what should this be like?

Reply:
I will repush and make it default